frontend/pages/redeem/[uuid]/thank-you.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Stack, Typography } from '@mui/material';
+import Lottie from 'lottie-react';
+import { NextPage } from 'next';
+import orangeTick from '../../../assets/orangeTickLottie.json';
+import { lottieSx, stackSx } from '../../../styles/redeem/thankYou';
+
+const ThankYou: NextPage = () => {
+  return (
+    <Stack component="main" maxWidth="md" spacing={2} sx={stackSx}>
+      <Box sx={lottieSx}>
+        <Lottie autoPlay loop={false} animationData={orangeTick} />
+      </Box>
+      <Typography variant="h1" textAlign="center">
+        Thank you for donating!
```

Comment:
```suggestion
        Your donation goes a long way in making the lives of others a little better!
```

Reply:
It feels a bit overkill with the 'transform lives...' text below.

 I feel the 'Thank you for you donation' is fine given the more impactful body text.